src/main/java/org/candlepin/subscriptions/metering/service/prometheus/config/PrometheusServiceConfiguration.java

```diff
@@ -61,7 +61,7 @@ public PrometheusService prometheusService(ApiProvider provider) {
 
   @Bean
   EventController prometheusEventController(EventRecordRepository repo) {
-    return new EventController(repo);
+    return new EventController(repo, null);
```

Comment:
Passing null here will cause NPE in the event controller wherever this configuration is being used. It should be properly instantiated.

Reply:
Done